javascript/packages/eslint-config-nebenan/test/valid/hooks.jsx

```diff
@@ -0,0 +1,7 @@
+const useEffect = () => {};
+
+export const useMagicalFriendship = (layoutRef) => (
```

Comment:
test invalid examples?

Reply:
I cannot test an invalid example for this rule. It should cover 'ignore effect deps' 

valid: all kind of variables can be used inside an effect without deps check
invalid: ???

Reply:
i think we could test things like call order of hooks etc. use this for inspiration: https://github.com/facebook/react/blob/master/packages/eslint-plugin-react-hooks/src/RulesOfHooks.js

Reply:
That's a rule we don't touch. We use the defaults from airbnb. I don't think it makes sense to test rules set in airbnb config. I would stick with our own set of rules we apply on top of that

Reply:
good point, then good to go as is